src/mappings/pool.ts

```diff
@@ -265,26 +260,28 @@ export function handleSwap(event: LOG_SWAP): void {
   swap.tokenIn = event.params.tokenIn
   swap.tokenInSym = poolTokenIn.symbol
   swap.tokenOut = event.params.tokenOut
-  swap.tokenOutSym = poolTokenOut.symbol;
+  swap.tokenOutSym = poolTokenOut.symbol
   swap.tokenAmountIn = tokenAmountIn
   swap.tokenAmountOut = tokenAmountOut
   swap.poolAddress = event.address.toHex()
+  swap.userAddress = event.transaction.from.toHex()
   swap.poolTotalSwapVolume = totalSwapVolume
+  swap.value = swapUsdValue
+  swap.swapFeeValue = swapUsdValue.times(pool.swapFee)
```

Comment:
I'm about to change the calculation here, now what i do is like saying:

- We have a total swap end value of $100, the swap fee is 10% so the swap fee value is $10
But i think the way it's calculated is more like:
- We have a total swap end value of $100, the swap fee is 10%, so the swap fee value is 100/1.1 = 90,9090909091 , 100 - 90,9090909091 = 9,0909090909

Are we on the same page here @mikemcdonald?